udev/systemd-udev.json

```diff
@@ -1,14 +1,39 @@
 {
     "name": "udev",
+    "config-opts": [
+        "-Drootprefix=/app",
+        "--sysconfdir=/app/etc",
+        "--localstatedir=/app/var"
+    ],
+    "cleanup": [
+        "/bin",
+        "/etc/systemd",
+        "/include/systemd",
+        "/lib/libsystemd.so*",
+        "/lib/libnss_myhostname.so*",
+        "/lib/libnss_systemd.so*",
+        "/lib/libnss_mymachines.so*",
+        "/lib/libnss_resolve.so*",
+        "/lib/pkconfig/libsystemd.pc",
+        "/lib/security/pam_systemd.so*",
+        "/lib/systemd",
+        "/share/bash-completion",
+        "/share/dbus-1/system-services",
+        "/share/doc",
+        "/share/factory",
+        "/share/locale",
+        "/share/man",
+        "/share/pkgconfig/systemd.pc",
+        "/share/polkit-1/actions",
+        "/share/systemd",
+        "/share/zsh",
+        "*.d"
+    ],
```

Comment:
This list is probably good, I would ask to look at `meson_options.txt` to see if some of them can be not built entirely though.